html-and-web/html-and-the-web_personal-website/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+    <body style="background-color:rgb(255, 165, 127);"></body>
+ 
+  
+
+    <h1>Rubens Kuhn</h1>
+    <p>The Noob of Coders: He may look retarded,</p>
+    <p>but can code the sh#t out of you while doing so!</p>
+    <p></p>
```

Comment:
empty paragraph, you can delete this to keep your code clean; again: line break with < br>